code/addons/interactions/package.json

```diff
@@ -43,6 +43,11 @@
       "require": "./dist/manager.js",
       "import": "./dist/manager.mjs"
     },
+    "./preset": {
+      "types": "./dist/preset.d.ts",
+      "require": "./dist/preset.js",
+      "import": "./dist/preset.js"
```

Comment:
This `import` condition and the one above it are not correct, they're pointing to cjs files.  But we don't generate an ESM version, so I did this just to get tests passing.  It should probably be fixed the right way by generating esm.